hack/build/docker/cdi-func-test-registry-populate/populate-registry.sh

```diff
@@ -102,10 +101,10 @@ function pushImages {
         IMAGENAME=${FILE//.}
         echo "building image "$IMAGENAME
         buildah bud -t $IMAGENAME":latest" $images"/"$IMAGEDIR"/"
-	error $retval	
+	error $?	
```

Comment:
Curious why using error function instead of `set -e`?

Reply:
Does it matter that much? I am perfectly happy either way

Reply:
Well, the script does a bunch of other stuff with no error checking.  And I think it would be preferable for the container to exit with an error than fail silently (that way functional tests will never run, right?).

Obviously, this is an improvement, and that's great.  And if you want to merge it now, that is a-ok with me.

Reply:
Okay, @annastopel could you do an additional PR with set -e?